app/views/layouts/application.html.erb

```diff
@@ -16,12 +16,13 @@
             <% if current_user %>
```

Comment:
Prefer `<% if user_signed_in? %>` to this. `if user_signed_in?` reveals the reason for the `if` block more clearly.